src/components/AddFundsInterstitial.js

```diff
@@ -242,7 +243,7 @@ const AddFundsInterstitial = ({ network }) => {
                 size="large"
                 weight="bold"
               >
-                􀍡 Other amount
+                {` 􀍡 ${lang.t('add_funds.interstitial.other_amount')}`}
```

Comment:
@skylarbarrera I started going through and adding back some of the English translations, and I noticed some of these spaces with strange Unicode characters. Based on the app UI, I'm guessing these are used to render icons, but I was wondering how I would know which icon each represents without checking the app UI manually? Or is it something else?

Reply:
so there's are icons that unfortunately dont display in most editors, you should handle them the same as above as their position shouldnt change with the translation

Reply:
if you need to move around you can copy them tho